### ISSUE TITLE
Can't login: No configs given

### ISSUE THREAD

**Issue by nickyromeijn** [mgp25/Instagram-API#2633]
### Before submitting an issue make sure you have:
- [x] [Searched](https://github.com/mgp25/Instagram-API/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [x] [Read the FAQ](https://github.com/mgp25/Instagram-API/wiki/FAQ)
- [x] [Read the wiki](https://github.com/mgp25/Instagram-API/wiki)
- [x] [Reviewed the examples](https://github.com/mgp25/Instagram-API/tree/master/examples)
- [x] [Installed the api using ``composer``](https://github.com/mgp25/Instagram-API#installation)
- [x] [Using latest API release](https://github.com/mgp25/Instagram-API/releases)

### Purpose of your issue?
- [ ] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [x] Question
- [ ] Other

```php
<?php
set_time_limit(0);
date_default_timezone_set('UTC');
require __DIR__.'/../vendor/autoload.php';
/////// CONFIG ///////
$username = 'xxxxxx';
$password = 'xxxxxx';
$debug = true;
$truncatedDebug = true;

$ig = new \InstagramAPI\Instagram($debug, $truncatedDebug, $storageConfig = [
    'storage'    => 'file',
    'basefolder' => '../settings',
]);
try {
    $ig->login($username, $password);
} catch (\Exception $e) {
    echo 'Something went wrong when logging in: '.$e->getMessage()."\n";
    exit(0);
}

```

Error Log/var_dump:

```php
POST:  https://b.i.instagram.com/api/v1/launcher/sync/
DATA: signed_body=e7036232f2f355151f1052e5816d8fd1b270e85073ecd162262089426aafc2eb.{"_csrftoken":"IFpClX28BusCoQyHkx8MZydsvz8fBDTg","id":"4cb87f5a-f38e-4063-8828-723d09246b0e","configs":""}&ig_sig_key_version=4
← 400 	 49B
RESPONSE: {"message": "No configs given", "status": "fail"}
```

---
### Describe your issue
I've followed the instructions but can't seem to get past the login bit of the app. It seems no-one ran into this issue before so here goes. 

When logging in the application returns a 400 with the body  
```{"message": "No configs given", "status": "fail"}```
I have no clue what this means. I couldn't find any way to provide login options to the lib. Am I missing something?

**Comment by 8lbiasian** [comment]
Same thing, with Old version and the New version

**Comment by davidrodma** [comment]
This problem is happening with new sessions. Logins with old sessions continue to work.

**Comment by nickyromeijn** [comment]
@davidrodma I don't have any old sessions though. I wanted to try this lib with a fresh account. Is the issue currently looked into? If no any suggestions from where it might be originating? I'm guessing it's missing some parameters when posting to https://b.i.instagram.com/api/v1/launcher/sync/. But then again I haven't really skimmed through the internals properly yet

**Comment by JRoy** [comment]
I'll look into this

**Comment by davidrodma** [comment]
@nickyromeijn This problem began to occur an hour ago. I also look for a solution to the problems. But it might be just a temporary problem on Instagram. But of course the mgp25 staff will check if the problem continues.

**Comment by nickyromeijn** [comment]
@davidrodma Ah i see, just bad timing on my part then :). Thanks for info and quick replies. 

@JRoy Great to hear! 👍

**Comment by nemanjaflck** [comment]
Hi guys. My PR fixes this problem. #2634

**Comment by JRoy** [comment]
Fixed in e79acc0

Thanks to @nickyromeijn for your report and @nemanjaflck for your contribution!

Cheers.

**Comment by Sanan-84** [comment]
require realpath('.').'/vendor/autoload.php';
use InstagramAPI\InstagramID;
\InstagramAPI\Instagram::$allowDangerousWebUsageAtMyOwnRisk=true;
$ig = new \InstagramAPI\Instagram();
try{$ig->login($username,$password);}
catch(\Exception $e){echo $e->getMessage()."\n";}

Return:
InstagramAPI\Response\LauncherSyncResponse: No configs given.
 pleace help me ((:
